src/sidebar/components/top-bar.js

```diff
@@ -17,16 +19,44 @@ const SortMenu = require('./sort-menu');
 const SvgIcon = require('./svg-icon');
 const UserMenu = require('./user-menu');
 
+/**
+ * Button for opening/closing the help panel
+ */
+function HelpButton({ onClick }) {
+  const isActive = useStore(
+    store =>
+      store.getState().sidebarPanels.activePanelName === uiConstants.PANEL_HELP
+  );
+  return (
+    <button
+      className={classnames('top-bar__btn top-bar__help-btn', {
+        'top-bar__btn--active': isActive,
```

Comment:
Huh. Interesting question that prompts me to think that we should re-examine the conventions you've linked to here. I've used the `.component__element--modifier` naming here, which is what I've been doing for all state-related styling for a while. Yes, technically, this is a state applied by JavaScript, but...I feel like the `is-<whatever>` naming hearkens back to older days in which JS would only interact with certain parts of the DOM, instead of being full-on in-control all the time. That is, technically, _all_ modifiers are applied by JS, right :) ?

/cc @robertknight

Reply:
We discussed this with voices and the dynamic state classnaming is still a valid pattern. I will make a follow-up PR to make these consistent in this component.